techniques/system/common/1.0/restart-services.cf

```diff
@@ -79,7 +79,7 @@ bundle agent restart_services
 
     # Ensure at least one syslog is running
     # We cannot detect which one is used for Rudder
-    !aix.!solaris::
+    rudder_reporting_syslog.!aix.!solaris::
```

Comment:
isn't it dealt with in the calling bundle ?

Reply:
We could rename the bundle to `restart_syslog` and only call it when syslog is used for reporting indeed, good idea.